tests/resources/testnet.yaml

```diff
@@ -1,69 +1,62 @@
-# Minimal config
-
-# Extends the minimal preset
-PRESET_BASE: 'minimal'
-
-# Free-form short name of the network that this configuration applies to - known
-# canonical network names include:
-# * 'mainnet' - there can be only one
-# * 'prater' - testnet
-# Must match the regex: [a-z0-9\-]
-CONFIG_NAME: 'minimal'
-
-# Transition
-# ---------------------------------------------------------------
-# TBD, 2**256-2**10 is a placeholder
-TERMINAL_TOTAL_DIFFICULTY: 115792089237316195423570985008687907853269984665640564039457584007913129638912
-# By default, don't use these params
-TERMINAL_BLOCK_HASH: 0x0000000000000000000000000000000000000000000000000000000000000000
-TERMINAL_BLOCK_HASH_ACTIVATION_EPOCH: 18446744073709551615
-
-
+# Extends the mainnet preset
```

Comment:
what testnet is this?

Reply:
It's actually copy of https://twisty-wednesday-4be.notion.site/ETH-CC-Testnet-Helder-fdc109a513cb44a68fe70f7038f2690b

Reply:
then i propose calling it `helder_testnet.yaml`

Reply:
Renamed in https://github.com/ChorusOne/eth-staking-smith/pull/45/commits/c7a9840952aece262d48c7735ee0f2e966dd36b4